.github/workflows/orm2_tests.yml

```diff
@@ -29,5 +26,7 @@ jobs:
       - run: composer validate
 
       - uses: "ramsey/composer-install@v2"
+        with:
+          dependency-versions: "lowest"
```

Comment:
Sorry, but it does not make sense to run all tests on PHP 7 against lowest deps only.